### ISSUE TITLE
IOS 12 White Topbar Issue, version 4.0.2

### ISSUE THREAD

**Issue by kdetry** [wix/react-native-navigation#5728]
### Issue Description

Topbar background color property is not working at RNN version 4.0.2 for IOS 12. It seems like white background.

There is no problem with IOS 13.1.

### Steps to Reproduce / Code Snippets / Screenshots

```js
Navigation.setDefaultOptions({
    topBar: {
        background: {
            color: "#fc272b"
        }
    },
});
```

There is not a problem at react-native-navigation version 3.6.0

---
### Environment
* React Native Navigation version: 3.6.0
* React Native version: 0.61.4
* Platform(s) (iOS, Android, or both?): IOS 12
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator and Device, IOS 12, Debug and Release

**Comment by sm2017** [comment]
I have same issue!

**Comment by sm2017** [comment]
Any solution?

**Comment by sm2017** [comment]
@guyca @yogevbd Can you take a look

**Comment by guyca** [comment]
This will be addressed in the next version, I believe it's already fixed in `4.0.4-snapshot.629`
